py3status/modules/spotify.py

```diff
@@ -150,18 +151,22 @@ def _get_text(self):
                 )
                 if playback_status.strip() == "Playing":
                     color = self.py3.COLOR_PLAYING or self.py3.COLOR_GOOD
+                    state = "playing"
```

Comment:
For ~~`state`~~`playback`, I'd like to pass along the value from `playback_status` instead of doing it like this (as it keeps things a bit more clean). Does `playback_status` print `Paused` too? And do we still need `strip()`?

Reply:
I will check `playback_status` and `strip()`.
Do you have strong preference for calling this placeholder `playback`? I have to admit I like `state` more (and I think it could be more consistent with other modules).
Also, I would like output from this placeholder to be lowercase if you don't mind it.

Reply:
>Do you have strong preference...

I have no preference, just that I like to keep things simple and code small... usually done by removing anything hardcoded and keeping source data untouched in favor of customizing them all in `format` instead. For instance...

If ...
```python
        playback_status = self.player.Get(
            "org.mpris.MediaPlayer2.Player", "PlaybackStatus"
        )
```
... gave `Playing` or `Paused`, then that's the value MPRIS2 gave us. Untouched.

You want lowercase, you can do this too.

`format = "\?if=playback=Playing playing"` (or use a symbol instead). Both should give us same thing.

>Do you have strong preference for calling this placeholder `playback`?

Yes, because I want to avoid generic words like `percent`, `state`, `count`, `code`, etc. Some modules can have more than one `percent`, various of `state` or `count`-related placeholders.

In this instance, I'd try to use nouns... `playback` (state), `volume` (percent), `task` (count), `battery` (count), `charging` (boolean), et cetera. I also think we can avoid long placeholders like `charging_state`, `playback_status`, `volume_percent`, `task_count`, etc in favor of smaller format strings.